style.css

```diff
@@ -1,8 +1,253 @@
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
 }
+
+html{
+  font-size: 16px;
+}
+
+body{
+  background-color: #fff;
+}
+
+.flex{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 0 1%;
+}
+
+
+.inner{
+  margin: 0;
+  padding: 1%; 
+}
+
+.container{
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  
+}
+
+.navigation{
+  width: 100%;
+  background-color: black;
+}
+
+.left-nav{
+  width: 70%;
+  align-items: center;
+  justify-content: flex-start;
+  position: sticky;
+}
+
+
+.logo img{
+  width: 140px;
+  padding: 3%;
+  margin-top: 1%;
+  cursor: pointer;
+}
+
+.navbar{
+  width: 85%;
+}
+
+.navbar a:hover {
+  text-decoration: underline;
+  text-decoration-color: red;
+}
+
+ul li{
+  color: #fff;
+  list-style: none;
+  font-size: 1rem;
+  margin-right: 40px;
+  cursor: pointer;
+}
+
+.logo{
+  width: 15%;
+}
+
+.search{
+  background-color: transparent;
+  border: 2px solid rgb(218, 20, 20);
+  border-radius: 8px;
+  color: #fff;
+  padding: 5px;
+  margin: 10px;
+  font-size: 1rem;
+  font-family: inherit;
+}
+
+.search:focus{
+  outline: 0;
+  background-color: transparent;
+}
+
+.search::placeholder{
+  color: #fff;
+}
+
+main{
+  display:flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 30px;
+}
+
+.movie {
+  width: 300px;
+  margin: 10px;
+  border-radius: 3px;
+  box-shadow: 0.2px 4px 5px rgba(0,0,0,0.1);
+  background-color: #fff;
+  position:relative;
+  overflow: hidden;
+}
+
+
+.movie img{
+  width:100%;
+}
+
+.movie-info{
+  color: black;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0.5rem 1rem 1rem;
+  letter-spacing: 0.5px;
+}
+
+.movie-info h3{
+  margin-top: 0;
+}
+
+.movie-info span{
+  background-color: var(--primary-color);
+  padding: 0.25rem 0.5rem;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.movie-info span.green{
+  color:lightgreen;
```

Comment:
Always prefer to use hex or rgb or hsl color values over named values like lightgreen or black